DataStructures/Lists/DoublyLinkedList.java

```diff
@@ -43,6 +49,7 @@ public DoublyLinkedList(int[] array) {
         for (int i : array) {
             insertTail(i);
         }
+        size = array.length;
```

Comment:
If size > 0 before this function is called, size != array.length

Reply:
use size += array.length

Reply:
Thank you @rbshealy  for reviewing, I will make the required changes.

Reply:
Yes this is correct, didn't realize this is the constructor. Great work!

Reply:
Thank you